libs/utilities/build.gradle

```
@@ -16,6 +16,8 @@ dependencies {
 
     implementation platform("net.corda:corda-api:$cordaApiVersion")
     implementation 'net.corda:corda-base'
+    implementation 'net.corda:corda-application'
+    implementation 'net.corda:corda-serialization'
```

```suggestion
    api 'net.corda:corda-application'
    api 'net.corda:corda-serialization'
```
since classes in question used in extension methods and as input types on method signature.

---

Fair enough, and also `:base` because of `ByteSequence` in:
```kotlin
fun ByteArray.toByteSequence() : ByteSequence {
    return OpaqueBytesSubSequence(this, 0, this.size)
}
```